src/test/java/expediaproductobjects/FilterOptions.java

```diff
@@ -0,0 +1,26 @@
+package expediaproductobjects;
+
+import org.openqa.selenium.WebDriver;
+
+public class FilterOptions extends BasePage{
+	
+	public String starRatingXpath = "//label[@for='f-star-rating-[RATING]']";
+	public String slider = "//div[@role='slider']";
```

Comment:
Try to follow the same naming convention

Reply:
Will fix